### ISSUE TITLE
Safari 13 support?

### ISSUE THREAD

**Issue by bukharin** [jitsi/lib-jitsi-meet#1713]
## Description

We investigated that users with Safari 13 version (no matter desktop or mobile) does not hearing other participants. Does LJM supports this browser?

It seems that in this browser `stream` is `undefined` [here](https://github.com/jitsi/lib-jitsi-meet/blob/d4bb376f96e50fa8c2951bd6428902f1ddeacaf3/modules/RTC/TraceablePeerConnection.js#L327)

# Environment details

Safari on MacOS 13.1, Safari Mobile 13.1

**Comment by saghul** [comment]
I don't think we do, @jallamsetty1 can you confirm?

**Comment by bukharin** [comment]
> I don't think we do, @jallamsetty1 can you confirm?

It seems that meet.jit.si allows user to launch conference in web on Safari 13

**Comment by saghul** [comment]
Yes, we may need to adjust our supported version so you are not allowed to join if we don't support that version.

**Comment by jallamsetty1** [comment]
We do support Safari 13. Safari 12.1 is when Safari switched to unified plan and we support all versions after that. 
@bukharin You said Safari 13 users don't hear other users, but do they see the other users ? Also are these users joining audio/video muted ? We know of an issue on iOS where users joining audio/video muted were not hearing other participants and we have merged a fix for this.

**Comment by bukharin** [comment]
@jallamsetty1 

> Also are these users joining audio/video muted ?

No. Start unmuted.

> We know of an issue on iOS where users joining audio/video muted were not hearing other participants and we have merged a fix for this.

We use latest master build of LJM with this fix

I found a user with iPhone 6 + Safari 12.1 and he has the same issue. He does not hear and see any other participants. But remote participants hears and see him.

We collect client exceptions. On Safari 13 the problem seems here:

```
TypeError
Right side of assignment cannot be destructured
```

In this lines of minified code

```
{snip}  device will be used instead",ve,y)}),Y}),this.getStreamID=({id:ve})=>ve,this.getTrackID=({id:ve})=>ve),this.pcConstraints=N.a.isChromiumBas
```

It seem that all remote tracks could not be correctly added in this Safari versions. And I guess that `stream` is `undefined` [here](https://github.com/jitsi/lib-jitsi-meet/blob/d4bb376f96e50fa8c2951bd6428902f1ddeacaf3/modules/RTC/TraceablePeerConnection.js#L784) for some reasons
